src/palace/manager/celery/tasks/generate_inventory_and_hold_reports.py

```diff
@@ -346,6 +339,72 @@ def inventory_report_query() -> Select:
             ),
             lic.c.checkouts_left.label("remaining_loans"),
             lic.c.terms_concurrency.label("allowed_concurrent_users"),
+        )
+        .select_from(LicensePool)
+        .join(Identifier, LicensePool.identifier_id == Identifier.id)
+        .outerjoin(isbn, Identifier.type != Identifier.ISBN)
+        .join(Edition, Edition.id == LicensePool.presentation_edition_id)
+        .join(Work, LicensePool.work_id == Work.id)
+        .join(DataSource, LicensePool.data_source_id == DataSource.id)
+        .join(Collection, LicensePool.collection_id == Collection.id)
+        .join(
+            IntegrationConfiguration,
+            Collection.integration_configuration_id == IntegrationConfiguration.id,
+        )
+        .join(
+            IntegrationLibraryConfiguration,
+            IntegrationConfiguration.id == IntegrationLibraryConfiguration.parent_id,
+        )
+        .join(Library, IntegrationLibraryConfiguration.library_id == Library.id)
+        .outerjoin(wg_subquery, Work.id == wg_subquery.c.work_id)
+        .join(collection_sharing, true())
+        .outerjoin(lic, true())
+        .where(
+            Library.id == bindparam("library_id"),
+            IntegrationConfiguration.id.in_(
+                bindparam("integration_ids", expanding=True)
+            ),
+        )
+        .order_by(
+            Edition.sort_title,
+            Edition.sort_author,
+            DataSource.name,
+            IntegrationConfiguration.name,
+        )
+    )
+
+
+def palace_inventory_activity_report_query() -> Select:
+    """A query for holds report with hold information."""
+
+    isbn = _best_isbn_lateral()
+    wg_subquery = _comma_separated_sorted_work_genre_list_subquery()
+    collection_sharing = _is_shared_collection_lateral()
+    lib_holds = _library_holds_lateral()
+    lib_loans = _library_loans_lateral()
+
+    return (
+        select(
+            Edition.title,
+            Edition.author,
+            Identifier.identifier,
+            func.coalesce(
+                case(
+                    (Identifier.type == Identifier.ISBN, Identifier.identifier),
+                    else_=isbn.c.identifier,
+                ),
+                "",
+            ).label("isbn"),
+            Edition.language,
+            Edition.publisher,
+            Edition.medium.label("format"),
+            Work.audience,
+            func.coalesce(wg_subquery.c.genres, "").label("genres"),
+            DataSource.name.label("data_source"),
+            IntegrationConfiguration.name.label("collection_name"),
+            LicensePool.licenses_available.label(
+                "total_library_allowed_concurrent_users"
```

Comment:
Minor: Should this label be more specific. I think its accurate, but it doesn't really capture that this number is the total minus currently used. Something like `available_library_concurrent_users` seems to capture this difference better for me. This may have come in from the requirements though, or maybe I'm not understanding the full context.

Reply:
This was the name requested in the ticket. And actually, I want to check in to ensure that this is what we really want here. More to come.

Reply:
Moving to using `LicensePool.licenses_owned` for `total_library_allowed_concurrent_users` and `library_hold_ratio` after reviewing how the accounting works. It is a much more stable denominator for the hold ratio.